language-support/hs/bindings/test/DA/Ledger/Sandbox.hs

```diff
@@ -40,12 +41,15 @@ sandboxProcess SandboxSpec{dar} = do
 startSandboxProcess :: SandboxSpec -> IO (ProcessHandle,Maybe Handle)
 startSandboxProcess spec = do
     processRecord <- sandboxProcess spec
-    (_,hOutOpt,_,proh) <-
+    (_,hOutOpt,Just hErrOpt,proh) <-
         createProcess processRecord {
         std_out = CreatePipe,
         std_err = CreatePipe, -- Question: ought the pipe to be drained?
```

Comment:
Why make this CreatePipe then drain it to stdout? Just pass a handle to std_err.

Reply:
oops debugging :)